cmd/opampsupervisor/supervisor/supervisor.go

```diff
@@ -49,6 +51,9 @@ var (
 
 	//go:embed templates/owntelemetry.yaml
 	ownTelemetryTpl string
+
+	lastRecvRemoteConfigFile     = "last_recv_remote_config.yaml"
+	lastRecvOwnMetricsConfigFile = "last_recv_own_metrics_config.yaml"
```

Comment:
```suggestion
	lastRecvRemoteConfigFile     = "last_recv_remote_config.proto"
	lastRecvOwnMetricsConfigFile = "last_recv_own_metrics_config.proto"
```
These are proto files, right?

Reply:
We are marshalling the protos and writing that content to file.

Reply:
Thinking again, maybe `.proto` isn't the right file extension here. But since we're marshalling them, shouldn't they be `.dat` or `.protobuf` instead of `.yaml` since they're not YAML files?

Reply:
Right, you are correct.